edge-modules/load-gen/Program.cs

```diff
@@ -45,7 +47,25 @@ static async Task Main()
                     try
                     {
                         await SendEventAsync(moduleClient, batchId, Settings.Current.TrackingId, messageIdCounter);
-                        //TODO: Report sending message successfully to Test Result Coordinator
+
+                        // Report sending message successfully to Test Result Coordinator
+                        await Settings.Current.TestResultCoordinatorUrl.ForEachAsync(async trcUrl =>
+                        {
+                            Uri testResultCoordinatorUrl = new Uri(
+                                trcUrl,
+                                UriKind.Absolute);
+                            TestResultCoordinatorClient trcClient = new TestResultCoordinatorClient { BaseUrl = testResultCoordinatorUrl.AbsoluteUri };
```

Comment:
>trcClient [](start = 56, length = 9)

nit: move trcClient before while

Reply:
`Settings.Current.TestResultCoordinatorUrl` has a type of `Option<string>` that's why I need the `ForEachAsync()` to extract the optional value `trcUrl`